cadc-download-manager-server/src/main/resources/META-INF/resources/applet.jsp

```diff
@@ -129,19 +151,28 @@ String bodyFooter = skin + "bodyFooter";
     
 <div style="padding-left: 2em; padding-right: 2em">
     <form action="/downloadManager/download" method="POST">
-        <%      for (URI uri: uriList) {
-            String uriStr = uri.toString();
+        <%      for (DownloadTuple tuple: tupleList) {
```

Comment:
For all of the `for` loops that have replaced the `c:forEach`, was it not possible to keep the latter due to the `df.format()` call?